src/index.ts

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env node
+import { cosmiconfigSync }  from "cosmiconfig";
+import program  from "commander";
 
 import { run } from "./runner";
+import { ConfigInterface } from "./config.interface";
 
-run();
+const cliConfig = program
+    .option('-p, --project [project]', 'TS project configuration file (tsconfig.json)')
+    .parse(process.argv)
+
+const moduleName = 'ts-prune';
+const explorerSync = cosmiconfigSync(moduleName);
+const fileConfig = explorerSync.search();
+
+if (!fileConfig && !cliConfig) {
+    throw new Error(`Config not found.
+    You can read about the configuration options here:
+    https://github.com/nadeesha/ts-prune#configuration
```

Comment:
I may be missing something. But I can't seem to follow this permalink?

Reply:
No worries this an run time error message for wrong configuration, which is pointing to main readme.md file, the link will start working normally after this mr would be merged, similar to this one https://github.com/nadeesha/ts-prune#getting-started

I have updated the readme file in this pr, and added configuration section, which this link is pointing to

Reply:
Removed this check for saving backward compatibility